docs/platforms/python/usage/set-level/index.mdx

```diff
@@ -7,3 +7,5 @@ sidebar_order: 5
 The level - similar to logging levels - is generally added by default based on the integration. You can also override it within an event.
 
 <PlatformContent includePath="set-level" />
+
+For a list of possible levels see the [API Documentation](https://getsentry.github.io/sentry-python/api.html#sentry_sdk.api.set_level)
```

Comment:
```suggestion
For a list of possible levels, see the [API Documentation](https://getsentry.github.io/sentry-python/api.html#sentry_sdk.api.set_level).
```

Reply:
Maybe I'm missing something, but I don't see a list of possible levels on that link?

Reply:
Not yet, we need to update the APIdocs. The changes for that are already in the our pipeline and well be released soon.